src/Progress.cpp

```diff
@@ -57,11 +70,13 @@ void Progress::incrementCounter() {
       if (_curr_interval >= _intervals.size())
         break;
     }
-#pragma omp flush (_counter, _curr_interval)
```

Comment:
this flush ought to be deleted, since flushes are implied at the end of critical sections